extensions/machine-learning-services/src/views/models/azureModelsTable.ts

```diff
@@ -36,9 +36,22 @@ export class AzureModelsTable extends ModelViewBase implements IDataComponent<Wo
 			.withProperties<azdata.DeclarativeTableProperties>(
 				{
 					columns: [
-						{ // Id
-							displayName: constants.modeIld,
-							ariaLabel: constants.modeIld,
+						{ // Name
```

Comment:
Make sure you're going through and doing a brief accessibility pass on all this UI you're adding (tab order and voice reader being the most important)

Reply:
sure, none of the UIs are done yet, will do that when the are stable 

---
In reply to: [385794063](https://github.com/microsoft/azuredatastudio/pull/9377#discussion_r385794063) [](ancestors = 385794063)